connect/clients/nats.py

```diff
@@ -103,6 +112,12 @@ async def nats_sync_event_handler(msg: Msg):
     data = msg.data.decode()
     logger.trace(f"nats_sync_event_handler: received a message on {subject} {reply}")
 
+    # Emit an app_sync message so LFH clients that are listening only for
+    # messages from this LFH node will be able to get all sync'd messages
+    # from all LFH nodes.
+    client = await get_nats_client()
+    await client.publish(nats_app_sync_subject, msg.data)
+
```

Comment:
These 2 lines emit the EVENTS.app_sync message from within the NATS sync event handler.  The premise is that the local NATS sync subscriber receives every sync message from all the configured LFH servers in the network, so instead of making a LFH client configure exactly the same sync connections, the client can listen for EVENTS.app_sync from the nats server in the local LinuxForHealth node and still get all the sync messages from all the LFH servers in the network.  This reduces configuration requirements and results in fewer network connections.